plugins/ruby-foreman-omaha/debian/gem.list

```diff
@@ -0,0 +1,3 @@
+# list of gem urls to download via Jenkins, space separated
+GEMS="https://rubygems.org/downloads/foreman_omaha-2.0.0.gem"
```

Comment:
You forgot to bump this to 3.0.0 so it failed

Reply:
Alright. Will fix that. So many different places to put stuff. Probably just lost sight of them all.